src/cheri-pte-ext.adoc

```diff
@@ -75,26 +107,28 @@ access that caused the update, and the sequence is interruptible. However, the
 hart must not perform explicit memory access before the PTE update is globally
 visible.
 
-NOTE: The behavior of the CW bit takes priority over the CD bit. Therefore,
-implementations must not take action to change or raise an exception related to
-the CD bit when the CW bit is clear.
+When CW is set, the CRG bit indicates the current generation of the virtual memory page with
+regards to the ongoing capability revocation cycle. Two schemes are permitted:
 
-[#menvcfg_pte,reftext="menvcfg PTE bits"]
-=== Extending the Machine Environment Configuration Register (menvcfg)
+* A load page fault exception is raised when a capability load or AMO instruction is executed,
+the authorizing capability grants <<c_perm>>
+and the virtual page's CRG bit does not equal senvcfg.CRG.
+* A load page fault exception is raised when a capability load or AMO instruction is executed,
+the authorizing capability grants <<c_perm>>,
+the capability read from memory has tag set
+and the virtual page's CRG bit does not equal senvcfg.CRG.
 
-The *menvcfg* register is extended to allow discovering whether the
-implementation supports the CD bit.
+NOTE: An implementation may avoid a dependency on the tag
+bit value of the capability read. This will introduce additional traps during revocation sweeps.
 
-The *menvcfg* register operates as described in cite:[riscv-priv-spec]. {cheri_base_ext_name} adds a new enable bit as shown in
-xref:menvcfgreg[xrefstyle=short] when XLEN=64.
+[#senvcfg_pte,reftext="senvcfg PTE bits"]
+=== Extending the Supervisor Environment Configuration Register (menvcfg)
```

Comment:
There is a typo here. This section is for `senvcfg` instead of `menvcfg`.

Reply:
yes - I'm fixing up various things - will commit that change soon.